armi/reactor/assemblies.py

```diff
@@ -1191,7 +1204,11 @@ def countBlocksWithFlags(self, blockTypeSpec=None):
         blockCounter : int
             number of blocks of this type
         """
-        return len(self.getBlocks(blockTypeSpec))
+        if blockTypeSpec is None:
```

Comment:
Not your fault, but can we fix the docstring in this method?  It says:

> Returns the block at a specified axial dimension elevation (given in cm).

But it looks like it should say:

> Returns the number of blocks at a specified axial dimension elevation (given in cm).

Also, we can just delete this text from the docstring:

> Used as a way to determine what block the control rod will be modifying with a mergeBlocks.